README.md

```diff
@@ -34,7 +85,8 @@ cp kubectl-crossplane /usr/local/bin
 #### Install the Platform Configuration
 
 ```console
-PLATFORM_CONFIG=registry.upbound.io/upbound/platform-ref-gcp:v0.0.4
+PLATFORM_VERSION=v0.0.4
```

Comment:
Can we update the version here?

Reply:
This is the latest available version in this repo: https://github.com/upbound/platform-ref-gcp/tags
Could you elaborate a bit what do you mean by updating?

Reply:
Since we are changing all the namespace secret references to `Upbound-system`, we should increment the version.

Reply:
I see, thanks!